nilearn/regions/signal_extraction.py

```diff
@@ -133,6 +135,22 @@ def _get_labels_data(
     dim : :obj:`int`, optional
         Integer slices mask for a specific dimension.
 
+    keep_masked_labels : :obj:`bool`, optional
+        When a mask is supplied through the "mask_img" parameter, some
+        labels in the atlas may not have any brain coverage within the
+        masked region, resulting in empty time series for those labels.
```

Comment:
you may want to take advantage of the `_utils.fill_doc` mechanism to avoid repeating this docstring